pkg/controller/endpoint/endpoints_controller_test.go

```diff
@@ -95,39 +95,43 @@ func makeTestServer(t *testing.T, namespace string, endpointsResponse serverResp
 
 type endpointController struct {
 	*EndpointController
-	podStore     cache.Store
-	serviceStore cache.Store
+	podStore       cache.Store
+	serviceStore   cache.Store
+	endpointsStore cache.Store
```

Comment:
Please add a test case that proves that we correctly wait for all informers to be synced.